src/datalogger/src/registry.rs

```diff
@@ -12,6 +12,7 @@ const SENSOR_NAMES: [&str; 9] = [
     "timed_switch_2",
     "ds18b20",
     "k30_co2",
+    "basic_evo",
```

Comment:
should this be "basic_evo_n2o".  there's many different basic evos. https://www.smartgas-cn.com/en/index.php/home/product/detail/aid/29.  or do they all use the exact same protocol, in which case it's up to the user to simply name the sensor correctly during `set`